inspector/app/renderer/lib/client-frameworks/js-oxygen.js

```diff
@@ -6,23 +6,41 @@ class JsOxygenFramework extends Framework {
     return 'js';
   }
 
+  get type () {
+    if (this.caps && this.caps.platformName) {
+      if (this.caps.platformName === 'windows') {
```

Comment:
Need `toLowerCase()` as `this.caps.platformName.toLowerCase()`?

Reply:
fixed